packages/functional-tests/src/app.ts

```diff
@@ -7,6 +7,8 @@ import * as MRESDK from '@microsoft/mixed-reality-extension-sdk';
 import * as MRERPC from '@microsoft/mixed-reality-extension-sdk/built/rpc';
 import AssetPreloadTest from './tests/asset-preload';
 import ClockSyncTest from './tests/clock-sync-test';
+import delay from './utils/delay';

```

Comment:
imports aren't sorted. does tslint not flag this for you?

Reply:
fixed